From 7966740fc34a8efd0fd5eb04084f58432a0f0526 Mon Sep 17 00:00:00 2001
From: Devyn Liu <liudingyuan@huawei.com>
Date: Sun, 28 Apr 2024 18:35:06 +0800
Subject: [PATCH 463/960] downstream: i2c: hisi: Correct the description
 comment for PIN_MUX METHOD

Correct the description comment for PIN_MUX METHOD and rename it
to hisi instead of dw.

Signed-off-by: Devyn Liu <liudingyuan@huawei.com>
Signed-off-by: Junhua Lu <lujunhua7@h-partners.com>
Signed-off-by: Slim6882 <15605196882@163.com>
Signed-off-by: chenyi <chenyi211@h-partners.com>
Signed-off-by: Liu Shixin <liushixin2@huawei.com>
---
 drivers/i2c/busses/i2c-hisi.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/i2c/busses/i2c-hisi.c b/drivers/i2c/busses/i2c-hisi.c
index 989ba813435c..68b1c116c5da 100644
--- a/drivers/i2c/busses/i2c-hisi.c
+++ b/drivers/i2c/busses/i2c-hisi.c
@@ -484,7 +484,7 @@ static void i2c_hisi_soft_reset(struct device *dev)
 }
 
 /**
- * i2c_dw_acpi_pin_mux_change - Change the I2C controller's pin mux through ACPI
+ * i2c_hisi_pin_mux_change - Change the I2C controller's pin mux through ACPI
  * @dev: device owns the SCL/SDA pin
  * @to_gpio: true to switch to GPIO, false to switch to SCL/SDA
  *
-- 
2.43.0

